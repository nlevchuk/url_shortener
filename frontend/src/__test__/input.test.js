import Input from '../input';

describe('<Input />', () => {
  it('should set onChange event', () => {
    const wrapper = shallow(
      <Input url='https://heroku.com' onChange={() => {}} readOnly={false} />
    );
    // console.log(wrapper.debug());
    expect(wrapper).toMatchSnapshot();
  });

  it('should set readOnly input', () => {
    const wrapper = shallow(
      <Input url='https://heroku.com' onChange={() => {}} readOnly={true} />
    );
    // console.log(wrapper.debug());
    expect(wrapper).toMatchSnapshot();
  });
});
