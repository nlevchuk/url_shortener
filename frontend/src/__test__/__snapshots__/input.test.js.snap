// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should set onChange event 1`] = `
<input
  onChange={[Function]}
  placeholder="Paste URL here"
  type="text"
  value="https://heroku.com"
/>
`;

exports[`<Input /> should set readOnly input 1`] = `
<input
  placeholder="Paste URL here"
  readOnly={true}
  type="text"
  value="https://heroku.com"
/>
`;
